readme.md

```diff
@@ -110,7 +110,7 @@ Once you search you should see a list similar to the one below (versions may be
 * [ViewExtensions](http://www.uwpcommunitytoolkit.com/en/master/helpers/ViewExtensions/)
 * [VisualTreeExtensions](http://docs.uwpcommunitytoolkit.com/en/master/helpers/VisualTreeExtensions/)
 * [WeakEventListener](http://docs.uwpcommunitytoolkit.com/en/master/helpers/WeakEventListener/)
-* [WebViewExtensions](http://docs.uwpcommunitytoolkit.com/en/master/helpers/WebViewExtensions/)
+* [WebView](http://docs.uwpcommunitytoolkit.com/en/master/helpers/WebView/)
```

Comment:
In terms of documentation, should we still append Extensions (e.g. WebView Extensions). Or if we decide to move everything under an Extensions group, that would work as well.